### ISSUE TITLE
How to get the generated .dot file?

### ISSUE THREAD

**Issue by jabbalaci** [EntilZha/pygraph#1]
If I want to get the generated .dot file (for instance for further, manual refining), how can I get it in a file?

**Comment by EntilZha** [comment]
Currently, there isn't a way to do it, but it looks like it would be simple to implement using http://graphviz.readthedocs.io/en/latest/api.html#graphviz.Graph.save and another command line flag. I'll take shot at implementing it real quick.

**Comment by jabbalaci** [comment]
I didn't dive in the source but I thought you created a temporary .dot file and you called dot / neato / etc. on it.

**Comment by EntilZha** [comment]
Turns out we are both right, I don't explicitly create a temporary dot file, but graphviz does. There is an option to pass into `render` http://graphviz.readthedocs.io/en/latest/api.html#graphviz.Graph.render which can instruct it to save or delete the source file.

In the last commit I added a flag `--dot` which will toggle the source save, by default it won't keep the source around but when `--dot` is present it will preserve the source. I didn't add anything to customize the name of the source file (currently its just the name of the image minus extension), but that wouldn't be hard to do if its important.

If it works for you I can do a quick release to PyPI later today
